app/layout.js

```diff
@@ -13,13 +13,18 @@ const geistMono = localFont({
 });
 
 export const metadata = {
-  title: "Create Next App",
+  title: "Advanced Routing",
   description: "Generated by create next app",
 };
 
 export default function RootLayout({ children }) {
   return (
     <html lang="en">
+      <head>
+        <meta name="viewport" content="initial-scale=1, width=device-width" />
+        <title>{metadata.title}</title>
```

Comment:
You don't actually need this, Next.js understands the metadata object just fine without these lines.